src/Umbraco.Core/Deploy/IDataTypeConfigurationConnector.cs

```diff
@@ -17,18 +15,18 @@ public interface IDataTypeConfigurationConnector
         IEnumerable<string> PropertyEditorAliases { get; }
 
         /// <summary>
-        /// Gets the environment-agnostic data type configurations corresponding to environment-specific configurations.
+        /// Gets the environment-agnostic preValues corresponding to environment-specific preValues.
```

Comment:
PreValues is not a concept and should also be updated in comments..

Reply:
Didn't seem like i got all of them. Should be fixed